packages/actor-query-result-serialize-sparql-json/lib/ActorQueryResultSerializeSparqlJson.ts

```diff
@@ -114,7 +112,12 @@ export class ActorQueryResultSerializeSparqlJson extends ActorQueryResultSeriali
           .map(([ key, value ]) => [ key.value, ActorQueryResultSerializeSparqlJson.bindingToJsonBindings(value) ])))}`;
           first = false;
           return res;
-        }).append(wrap(end(() => `\n]}${this.emitMetadata ? `,\n"metadata": { "httpRequests": ${this.httpObserver.requests} }` : ''}}\n`))),
```

Comment:
This is split to multiple lines to make it more readable for a developer, now with added condition checks.